src/utils/formValidation.ts

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
NIT: This may become a big mix of many kind of validations. Could we eventually split those in folders like : `utils/form-validation/addresses-validation.ts` ?

Reply:
Sure, once we have more stuff in it we could do that!